app/app.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require('./app.js')
+const request = require('supertest')
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed.js')
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data/index.js')
+
+
+
+
+beforeEach(() => seed({articleData, commentData, topicData, userData}));
```

Comment:
because you are using every property, this could be imported in as a whole object and given to the `seed` function without being deconstructed if you like